public/docs/ts/latest/guide/component-communication.jade

```diff
@@ -0,0 +1,707 @@
+  1. The assigned hero is notified about winning the job, the others receive a message about losing the opportunity.
+  
+  Here is a screenshot of our complete application:
+  
+figure.image-display
+  img(src="/resources/images/devguide/component-communication/app-ui-snapshot.png" alt="UI in action")
+  
+:marked
+  ### Live Example
+  
+  The application is available as a live example. It is a single Angular app that uses the router configured with separate routing paths 
+  for each scenario. To try the sample code of a specific scenario, use the tabs right beneath the __Component Communication__ title.
+  
+  [Live Example](/resources/live-examples/component-communication/ts/plnkr.html)
+  
+  Let's start, and dive into the details!
+  
+  ## #1: Parent to child communication with input property binding
+  
+  Angular allows components to declare _data-bound input properties_. These are like regular component, but the value of an 
+  input property takes its value from the expression bound to it. The framework automatically updates data-bound input properties
+  during change detection.
+  
+  This is a _one-way binding_. As the source value changes, the destination property of the component is changed, but it does not 
+  happen in the opposite direction: altering the value of the input property within the component does not affect the source.
+  
+  Input data binding can be used to pass data from a parent component to a single child, to all or a set of its children.
+  As its name suggests, we can pass only data values with input property binding. This method is not suitable for sending messages that can be represented only
+  with method calls, and this mechanism does not allow passing data from a child component to its parent.
+  
+  ### Establishing an input property binding
+  
+  To create an input property binding, we need to carry out these steps:
+  1. Adding an input property to the child component (apply the `@Input()` decorator on the particular property)
+  1. Using this property with the _input binding syntax_ in the parent template to bind it to an expression
+  
+  The Hero Job Board application uses input data binding to pass the information about an invited hero from `HeroJobBoard` (parent) to all
+  `HeroPanel` children. The `HeroJobBoard` component is the one that knows the circle of heroes who should be invited to listen to job requests, and it embeds 
+  a `HeroPanel` component instance for each hero.
+  
+  In this scenario, `HeroPanel` declares an input property, `hero`, as the `@Input()` decorator suggests:
+  
++makeExample('component-communication/ts/app/invite-heroes/hero-panel.ts', null, 'invite-heroes/hero-panel.ts', {pnk: /(@Input\(\))/g})
+
+:marked
+  `HeroJobBoard` can iterate through its `invitedHeroes` collection and set up its `HeroPanel` children so that a single hero is
+  assigned to each of them through the `hero` input property. As we learned before, the Angular property binding syntax 
```

Comment:
Please avoid passive constructions such as "is used".

Keep it simple:

We use Angular property binding to pass the current iteration hero to the child component.